services/movie_session.py

```diff
@@ -0,0 +1,58 @@
+import datetime
+from django.core.exceptions import ObjectDoesNotExist
+from db.models import MovieSession, CinemaHall, Movie
+from django.db.models.query import QuerySet
+
+
+def create_movie_session(
+        movie_show_time: datetime,
```

Comment:
movie_show_time is module?
Because you use `import datetime` not `from datetime import datetime`